src/main/java/me/ghosttypes/orion/modules/main/AnchorAura.java

```diff
@@ -257,12 +257,12 @@ private BlockPos findBreakPos(BlockPos targetPos) {
         return null;
     }
 
-    private boolean getDamagePlace(BlockPos pos) {
-        return placeMode.get() == Safety.Suicide || DamageUtils.bedDamage(mc.player, Utils.vec3d(pos.add(0.5, 0.5, 0.5))) <= maxDamage.get();
+    private boolean getDamagePlace(BlockPos pos) { // Thanks Crosby :)
```

Comment:
:raised_eyebrow: elaborate please

Reply:
Crosby told me how to fix the issue on discord 💀

Reply:
Bro did not, he told me

Reply:
![Screenshot_20230416-195744](https://user-images.githubusercontent.com/99092380/232350923-fb28363f-26db-4778-8cfd-fe022513c8e4.png)

Reply:
> Bro did not, he told me

shut up i told both of yall